homeassistant/components/zwave_js/discovery.py

```diff
@@ -100,9 +102,10 @@ class ZWaveDiscoverySchema:
             "Setback Thermostat",
             "Thermostat General",
             "Thermostat General V2",
+            "Setpoint Thermostat",
         },
-        command_class={CommandClass.THERMOSTAT_MODE},
-        property={"mode"},
+        command_class={CommandClass.THERMOSTAT_SETPOINT},
```

Comment:
I'm not so sure about this. I know from working with climate support in OZW before there are different types.
So this should probably be a second discovery scheme ?
I believe @raman325 created the initial climate support so he could check if this change will break his device.

Reply:
You're probably right. I reverted that change and added a separate schema.